bin/job

```diff
@@ -59,8 +59,9 @@ begin
   end
 rescue Interrupt
   if Kernel.const_defined?(:Paint)
-    $stderr.puts "\n#{Paint['WARNING', :underline, :yellow]}: Cancelled by user"
+    $stderr.print "\n#{Paint['WARNING', :underline, :yellow]}: Cancelled by user"
```

Comment:
Why the change from `puts` to `print`?

Reply:
> See for full details:
> https://trello.com/c/9sWZKxne/282-ttypager-breaks-the-terminal-on-interrupt#comment-60884aa45fc8de123ca1e421

tl;dr The `raise SignalException.new(2)` causes an additional line to be emitted. So switching to `print` accounts for it.

Reply:
```
$ irb
irb(main):001:0> raise SignalException.new(2)

$
```